packages/vite/src/index.ts

```diff
@@ -14,12 +15,16 @@ import { warnOnLocalBuilds } from './local-build'
 
 type Options = ViteOptions | ((config: DynamicConfiguration) => (ViteOptions | Promise<ViteOptions>))
 
-export default async function plugin(options: Options = {}) {
-	const config = await loadConfiguration()
-	const resolvedOptions = typeof options === 'function'
+export default async function plugin(options: Options = {}, laravelPath = process.cwd(), basePath = process.cwd()) {
```

Comment:
I'd rather have the two new arguments in an object instead, that can avoid a breaking change later.